internals/daemon/api_notices.go

```diff
@@ -116,7 +190,26 @@ func sanitizeTypesFilter(queryTypes []string) ([]state.NoticeType, error) {
 	return types, nil
 }
 
+func isAdmin(userID, daemonUID uint32) bool {
```

Comment:
To make it super-obvious what "userID" refers to:

```suggestion
func isAdmin(requestUID, daemonUID uint32) bool {
```